tests/spec/core/inlines-spec.js

```diff
@@ -245,18 +249,22 @@ describe("Core - Inlines", () => {
         <aside class="example" id="example-aside_thing" title="aside"></aside>
         <pre class="example" id="example-pre" title="pre">
         </pre>
+        <section data-include="${generateDataIncludeUrl(
+          `## Example Dynamic\nA dynamically generated heading`
+        )}" data-include-format="markdown"></section>
       </section>
       <p id="output">
         [[[#section]]]
         [[[#figure]]]
         [[[#example-aside_thing]]]
         [[[#example-pre]]]
+        [[[#example-dynamic]]]
         [[[#does-not-exist]]]
       </p>`;
     const doc = await makeRSDoc(makeStandardOps(null, body));
     const anchors = doc.querySelectorAll("#output a");
-    expect(anchors).toHaveSize(4);
-    const [section, figure, exampleAside, examplePre] = anchors;
+    expect(anchors).toHaveSize(6);
```

Comment:
What's the 6th one?

Reply:
`#does-not-exist`. Earlier it was converted into a span (in core/inlines).